examples/thermomechanics/inputs/thermal_crack.json

```diff
@@ -11,6 +11,7 @@
     "horizon"                       : {"value": 3.0e-4, "unit": "m"},  
     "final_time"                    : {"value": 7.5e-4, "unit": "s"},
     "timestep"                      : {"value": 7.5E-9, "unit": "s"},
+    "thermal_subcycle_steps"        : {"value": 100},
```

Comment:
Thermal crack does not solve heat transfer. Why do we need "thermal_subcycle_steps"?

Reply:
OK, makes sense.